### PR TITLE
Add selinux info

### PR DESCRIPTION
**Pull request by kolenic-martin**
This PR adds information regarding selinux permissions for docker images in the airnode-examples readme.

### CODE REVIEW

**Review by Siegrift — APPROVED** [review]
👍 LGTM.

File: packages/airnode-examples/README.md [hunk 1]
```diff
@@ -66,6 +66,10 @@ The following instructions will guide you step by step through the RRP flow. Ple
 depending on how you would like to run the example - e.g. you don't need to start a hardhat network locally when you
 intend to run the example with a public network.
 
+If using a Linux distribution that enforces Selinux policies, make sure allow the Docker images access to the host
```
Comment (aquarat) [inline packages/airnode-examples/README.md:69]:
```suggestion
If using a Linux distribution that enforces SELinux policies, make sure allow the Docker images access to the host
```

File: .changeset/eleven-books-shout.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@api3/airnode-examples': patch
+---
+
+Added Selinux information to airnode-examples readme.
```
Comment (aquarat) [inline .changeset/eleven-books-shout.md:5]:
```suggestion
Added SELinux information to airnode-examples README.
```

**Comment by aquarat** [comment]
I was floating past and struggled to stop myself :pray: . SELinux is usually capitalised as SELinux (Security Enhanced Linux).

**Pull request merged**